rca/home/models.py

```diff
@@ -259,6 +266,8 @@ def get_context(self, request, *args, **kwargs):
         context["news_and_events"] = get_news_and_events()
         context["alumni_stories"] = get_alumni_stories()
         context["hero_colour"] = LIGHT_HERO
+        context["tap_widget_code"] = mark_safe(self.tap_widget.script_code)
```

Comment:
**Issue**
I think this will throw an attribute error if no tap widget is set, as tap_widget here could be a NoneType if not set

Reply:
Agreed I missed that one. I have added the if into place